samples/test/features/identifier-first-auth.feature

```diff
@@ -5,6 +5,11 @@ Feature: Login with Identifier First
       And an App that assigned to a test group
       And a Policy that defines "Authentication"
       And with a Policy Rule that defines "Any one factor"
+      And a Policy that defines "Profile Enrollment"
```

Comment:
Technically ID-First login doesn't require password. Hence added this policy to make sure a user can login even with this policy.